resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -358,6 +358,12 @@
           description: >
             searches for at most X distinct, valid alignments for each read. The search terminates when it can't find more distinct valid alignments, or when it finds X, whichever happens first.          default: 5
           default: 5
+    - name: threads
```

Comment:
Set this up in the process requirements section. Access the value set for a process from a `{{ requirements.resources.cores }}` variable